### PR TITLE
fix(client): adjust mobile view for email-signup page

### LINKED ISSUE #41062: Fixing css to enhance the redability for user
In email signup page text is not structured refer to that attached images
If we add
"div class="intro-description" style="
    padding:0 15px;
""
it would look much good 
Before & after images 

Screenshot (142)

Screenshot (138)

### PR DESCRIPTION
**Pull request by Aanandgupta**
In this Pull Request, we are adding padding left/right 15px in intro-description in intro.css for email-sign-up page so that content is more readable in  singup page and which in turn fixes issue #41062

Images attached tells how it will look before and after the fix

Screenshot (142)

Screenshot (138)

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #41062

### CODE REVIEW

File: client/src/components/Intro/intro.css [hunk 1]
```diff
@@ -24,6 +24,10 @@
   border: none;
 }
 
+.intro-description{
```
Comment (RandellDawson) [inline client/src/components/Intro/intro.css:27]:
```suggestion
.intro-description {
```

Comment (RandellDawson) [reply client/src/components/Intro/intro.css:27]: @Aanandgupta The linter is expecting the CSS property name to have a space after it and the `{`.

**Comment by nhcarrigan** [comment]
This looks to be a duplicate of #41087

File: client/src/components/Intro/intro.css (hunk 1 above)
Comment (Aanandgupta) [reply client/src/components/Intro/intro.css:27]: Okyy sir will do it

**Comment by Aanandgupta** [comment]
> This looks to be a duplicate of #41087

I wrote down the code that would resolve the issue.but someone else made the pr after seeing it I was not knowing that.

**Comment by raisedadead** [comment]
@Aanandgupta can you fix the linting errors? If you need any help please reach out to us in the contributors chat.

**Comment by Aanandgupta** [comment]
> @Aanandgupta can you fix the linting errors? If you need any help please reach out to us in the contributors chat.

Hii @raisedadead ya sure I will be giving it a try.

**Review by moT01 — CHANGES_REQUESTED** [review]
Looks like there are two PR's open to fix this issue. I left a review on [the other one](https://github.com/freeCodeCamp/freeCodeCamp/pull/41087). This one has the same problems that I think should be changed.

**Review by raisedadead — DISMISSED** [review]
_@moT01 's original comment:_ https://github.com/freeCodeCamp/freeCodeCamp/pull/41087#pullrequestreview-592412653

This first image is this PR at about 730px or so I think:

Screen Shot 2021-02-17 at 10 40 29 AM

**The padding doesn't get applied until the media query at 500px.**

This second image is the intro for the curriculum:
Screen Shot 2021-02-17 at 10 41 33 AM

**It adds an extra 15px of padding here. I think this PR should change the email page to mimic whatever is on the curriculum intro page.** 

Some more digging shows that bootstrap is adding a negative margin to the `row`. Perhaps just removing that will give the page the correct padding at all widths.

Screen Shot 2021-02-17 at 10 51 01 AM

**Comment by raisedadead** [comment]
@Aanandgupta are you able to work through the proposed changes here, once again if you need help please reach out to us in the chat.

**Comment by Aanandgupta** [comment]
> @Aanandgupta are you able to work through the proposed changes here, once again if you need help, please reach out to us in the [contributors chat room](https://chat.freecodecamp.org/channel/contributors).

Hii @raisedadead  i have tested for different screen widths and observed that as mentioned above doing margin zero for classname row will do the job I have generated Pr for the same 
Thankyou

File: client/src/components/Intro/intro.css [hunk 2]
```diff
@@ -24,6 +24,10 @@
   border: none;
 }
 
+.row {
+  margin:0;
+}
+
```
Comment (nhcarrigan) [inline client/src/components/Intro/intro.css:30]:
```suggestion
.row {
  margin: 0;
}

```

Should resolve the linting issue.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: client/src/components/Intro/intro.css (hunk 2 above)
Comment (Aanandgupta) [reply client/src/components/Intro/intro.css]: > Should resolve the linting issue.
Hi @nhcarrigan 
Done the changes 
ThankYou

**Comment by Aanandgupta** [comment]
@nhcarrigan  please review this

**Review by moT01 — CHANGES_REQUESTED** [review]
Screen Shot 2021-02-23 at 10 58 15 AM

Screen Shot 2021-02-23 at 11 01 31 AM

It may be a little tough to see, but this setting takes away the negative margin on all `.row` elements across the site it seems. The "as seen in" banner in the second image, for instance, extends the whole width of the window on production, but has a margin on the PR (right side of image). It really minor, and looks good either way - but I think we should add some specificity to the css selector so it only targets the specific row on the `email-sign-up` page. This will make it so there's no unintended visual adjustments elsewhere.

Edit: I took a quick look @Aanandgupta, I think you could move the setting to the `email-sign-up.css` file and target the row with `.email-sign-up .row`. That should work I think.

**Comment by ahmadabdolsaheb** [comment]
Adding custom css should not be necessary as we are using the same grid system across our platform. 

We should nest `<IntroDescription />` in a `<Col>` like the following:
```js
  <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
    <IntroDescription />
    <strong>{t('misc.quincy')}</strong>
    <Spacer />
    <p>{t('misc.email-blast')}</p>
    <Spacer />
  </Col>
```

Also while at it, you could add a spacer under the grid to give a little space on the top:

```js
  <Grid className='default-page-wrapper email-sign-up'>
    <Spacer />
```

**Comment by raisedadead** [comment]
Hi @Aanandgupta are you still working on this?

**Comment by Aanandgupta** [comment]
Hii
So many different solutions i am bit confused to select the approach can u
please tell i should go with which of the solution

**Comment by ojeytonwilliams** [comment]
@Aanandgupta Ahmad's solution makes the most sense to me.  I recommend trying to get that working.

**Comment by nhcarrigan** [comment]
Hey @Aanandgupta - just checking in to see if you are finding success in implementing Ahmad's solution.

**Comment by raisedadead** [comment]
Seems like the OP is unavailable. Closing, @ahmadabdolsaheb do you mind creating a fresh PR.

**Pull request closed without merging**